src/jdom/bus.ts

```diff
@@ -282,16 +280,16 @@ export class JDBus extends JDNode {
         return this.host.log
     }
 
-    set minConsolePriority(priority: ConsolePriority) {
-        if (priority !== this._minConsolePriority) {
-            this._minConsolePriority = priority;
+    set minLoggerPriority(priority: LoggerPriority) {
+        if (priority !== this._minLoggerPriority) {
+            this._minLoggerPriority = priority;
             this.emit(CHANGE)
         }
     }
 
     private async pingLoggers() {
-        if (this._minConsolePriority < ConsolePriority.Silent) {
-            const pkt = Packet.packed(CMD_CONSOLE_SET_MIN_PRIORITY, "i", [this._minConsolePriority]);
+        if (this._minLoggerPriority < LoggerPriority.Silent) {
+            const pkt = Packet.jdpacked<[LoggerPriority]>(0x2000 | LoggerReg.MinPriority, "i8", [this._minLoggerPriority]);
```

Comment:
```suggestion
            const pkt = Packet.jdpacked<[LoggerPriority]>(0x2000 | LoggerReg.MinPriority, "i32", [this._minLoggerPriority]);
```

(thought it doesn't matter much)